packages/gollm/common/prompts/dataset_enrichment.py

```diff
@@ -1,36 +1,38 @@
-DATASET_ENRICH_PROMPT = """
+START_ENRICH_PROMPT = """
 You are a helpful agent who is an expert in mathematical modelling and epidemiology. Your job is to enrich a dataset with metadata by inspecting statistics of the dataset.
 You may have access to a document that relates to the given dataset. Use this document to determine sources, considerations for use, and additional information.
 
 Focus on providing detailed and informative descriptions of the dataset and its columns. Do not make up information, only use the information provided in the document and dataset stats.
 Focus on trying to determine which columns represent spatial or temporal data, and provide a detailed description of the data in these columns.
+"""
+
+DOCUMENT_EXTRACTION = """
+Try to extract information for sourceData, considerationsForUsingTheData, and additionalInformation from the document.
+"""
 
-Try to extract information for sourceData, considerationsForUsingTheData, and additionalInformation from the document. Do not make up information. If you cannot find the information, leave a field empty.
+NO_DOCUMENT_EXTRACTION = """
+sourceData, considerationsForUsingTheData, and additionalInformation should not be filled in for this task.
+"""
 
+DATASET_PROMPT = """
 Use the following dataset statistics as a reference:
 
 ---START DATASET---
 {dataset}
 ---END DATASET---
+"""
 
+DOCUMENT_PROMPT = """
 Use the following document as a reference:
 
----START DOCUMENT---
-{research_paper}
----END DOCUMENT---
+---DOCUMENT START---
+{document}
+---DOCUMENT END--
 """
 
-DATASET_ENRICH_PROMPT_WITHOUT_DOCUMENT = """
-You are a helpful agent who is an expert in mathematical modelling and epidemiology. Your job is to enrich a dataset with metadata by inspecting statistics of the dataset.
-
-Focus on providing detailed and informative descriptions of the dataset and its columns. Do not make up information, only use the information provided in the dataset stats.
-Focus on trying to determine which columns represent spatial or temporal data, and provide a detailed description of the data in these columns.
-
-sourceData, considerationsForUsingTheData, and additionalInformation should not be filled in for this task.
-
-Use the following dataset statistics as a reference:
-
----START DATASET---
-{dataset}
----END DATASET---
+DO_NOT_HALLUCINATE = """
+Do not hallucinate. Do not make up information. Only use the information provided in the dataset and the document.
```

Comment:
🤣

Reply:
LOL.
Could we also add the three laws of robotics.